src/patternfly/components/TreeView/tree-view-node.hbs

```diff
@@ -1,21 +1,32 @@
 {{#> tree-view-content}}
-  <{{# if tree-view-node--type}}{{tree-view-node--type}}{{else}}button{{/if}} class="pf-c-tree-view__node{{#if tree-view-node--modifier}} {{tree-view-node--modifier}}{{/if}}{{#if tree-view-node--IsActive}} pf-m-active{{/if}}"
+  <{{# if tree-view-node--type}}{{tree-view-node--type}}{{else}}button{{/if}} class="pf-c-tree-view__node{{#if tree-view-node--modifier}} {{tree-view-node--modifier}}{{/if}}"
     {{#if tree-view-node--attribute}}
       {{{tree-view-node--attribute}}}
     {{/if}}>
     {{#if tree-view-node--IsToggle}}
       {{> tree-view-node-toggle-icon}}
     {{/if}}
     {{#if tree-view-node--HasCheckbox}}
-      {{> tree-view-node-check}}
+      {{> tree-view-node-toggle}}
```

Comment:
Is this correct? Is a toggle always present if `HasCheckbox`?

Reply:
Nice find! I dont know why I wrote that, removing this and changing the line above to be `HasToggle` and then showing the toggle button